framework/src/components/storage/adapters/pgp_adapter.js

```diff
@@ -88,7 +88,7 @@ class PgpAdapter extends BaseAdapter {
 
 		monitor.attach(this.pgpOptions, this.options.logEvents);
 		monitor.setLog((msg, info) => {
-			this.logger.log(info.event, info.text);
+			this.logger.debug(info.event, info.text);
```

Comment:
Shouldn't this be `info`? As in framework/src/controller/application.js#341

Reply:
I think info level is more for `user` so DB related stuff should be almost always `debug`